### PR TITLE
Added tic-tac-toe

### PR DESCRIPTION
**Pull request by anayra-c6b9**
# Pull Request Template

**Feature Name:** tic-tac-toe

**Changes made:**<br>
Added tic-tac-toe project

### Self Check(Tick After Making pull Request)

- [x] I have read and followed the [Contribution Guidelines](/CONTRIBUTING.md)
- [x] This issue was assigned to me.
- [x] One Change in one Pull Request
- [x] My file is in the proper folder

- [x] PR is regarding Code
- [x] I am following clean code and Documentation and my code is well linted with flake8.
- [x] I am using external libraries and I have created requirements.txt in my project folder
- [x] Script is tested and running perfectly fine on my system

- [x] PR is regarding the documentation

> If the issue was not assigned to you Please don't make a PR, It will be marked as invalid.

Discuss in discord server:- http://pragmaticprogrammer.in/discord

## acknowledgment
- [x] I have tested my code and distributing it under GPL-3.0 License

### CODE REVIEW

File: Projects/count-down/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# COUNT-DOWN TIMER
```
Comment (Punit-Choudhary) [inline Projects/count-down/README.md:1]: Why is the count-down timer mentioned when the specified Issue and PR are for Tic-Tac-Toe?

File: Projects/count-down/main.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    print("Ring Ring, it's time!")
+
+# Ask the user to enter the countdown period in seconds
+
+
+t = int(input("Enter the time in seconds: "))
+
+# function call
+
+countdown(t)
```
Comment (Punit-Choudhary) [inline Projects/count-down/main.py:29]: Why is the count-down timer code here when the specified Issue and PR are for Tic-Tac-Toe?

**Review by Punit-Choudhary — CHANGES_REQUESTED** [review]

**Comment by Punit-Choudhary** [comment]
@anayra-c6b9 We are following the One-Change Per Issue/PR
Please remove the unnecessary committed files.

File: Projects/count-down/main.py (hunk 2 above)
Comment (anayra-c6b9) [reply Projects/count-down/main.py]: I'm sorry for that. Please review my project now.

**Pull request merged**